### PR TITLE
Provide guidance on TLS ca_file directives

### PR DESCRIPTION
**Pull request by philpennock**

### CODE REVIEW

**Review by matthiashanel — COMMENTED** [review]
Add to this page as well? https://docs.nats.io/nats-server/configuration/securing_nats/tls

File: nats_server/tls.md [hunk 1]
```diff
@@ -57,4 +57,22 @@ tls: {
   # clients will fail to connect (value is too low)
   timeout: 0.0001
 }
-```
\ No newline at end of file
+```
+
+### Certificate Authorities
```
Comment (matthiashanel) [inline nats_server/tls.md:62]: Link to this on line 8

File: gateways/gateway.md [hunk 2]
```diff
@@ -99,3 +99,19 @@ gateway {
 	]
 }
 ```
+
+#### `TLS` Entry
```
Comment (matthiashanel) [inline gateways/gateway.md:103]: link on line 71

**Review by matthiashanel — CHANGES_REQUESTED** [review]

**Comment by philpennock** [comment]
This is the wrong repo.  Reproduced in https://github.com/nats-io/nats.docs/pull/163 and I tried to accommodate Matthias's points above.

**Pull request closed without merging**